docs/data/base/getting-started/installation/installation.md

```diff
@@ -1,23 +1,25 @@
 # Installation
 
-<p class="description">Install MUI Base, a robust library of blank ("unstyled") React UI components and low-level hooks.</p>
+<p class="description">Install MUI Base, a library of headless ("unstyled") React UI components and low-level hooks.</p>
 
-## npm
+## Default installation
```

Comment:
Why "default"? Is there any other?
Perhaps "npm package installation" could be a better title?

Reply:
That's a good question. "Default installation" here is my attempt to standardize the headers on all of these pages. It makes more sense in the context of the other libraries, because they have alternative/additional installation instructions involving styled-components, CDNs, icons, etc.

For Base, since it only has the one package to install, we could just remove this header entirely. It would break from the pattern but it makes sense to do so here.